macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -88,6 +90,18 @@ open class AvatarView: NSView {
 			initialsTextField.backgroundColor = avatarBackgroundColor
 		}
 	}
+	
+	/// The initials font color of the avatar view when no image is provided.
+	/// Setting this property will override the default provided color.
+	@objc open var initialsFontColor: NSColor {
```

Comment:
we already allow them to override background color, any client who wants to customize background color probably needs to customize font color too for accessibility.

Reply:
For the coauthoring scenario I know that the background color needs to be customized so that each coauthor gets the color matching the canvas presence indicator. I assume the foreground color may need to be changed if they do in the same direction of tint/shade combinations for both text and background colors?

Reply:
for coauthor scenario, we should have "ring" color like ios though right? is this something office is currently using?

Reply:
I have created a different task to follow up on parity with iOS avatar view.